metabase/metabase
# Show lighthouse illustration toggle

Epic https://github.com/metabase/metabase/issues/22826

How to test toggle:
- Run Metabase EE
- Go to Admin > Settings > Appearance 
- Enable/disable the lighthouse illustration
- Make sure it works both for home and login pages

How to test the illustration with a custom brand color:
- Run Metabase EE
- Go to Admin > Settings > Appearance 
- Enable the lighthouse illustration
- Change the brand color and refresh the page
- Make sure both home and login pages show the illustration in the custom brand color

<img width="571" alt="Screenshot 2022-07-12 at 18 19 24" src="https://user-images.githubusercontent.com/8542534/178527514-166f0717-0f64-44dc-be10-ffc3b42fc5ef.png">

<img width="1728" alt="Screenshot 2022-07-12 at 18 32 14" src="https://user-images.githubusercontent.com/8542534/178528342-9a8a1531-e340-4c8f-9a05-41a6f38266dd.png">
